frontend/src/app/pages/exercises/exercise/shared/utility/parse-csv.ts

```diff
@@ -0,0 +1,311 @@
+import type {
+    ConditionParameters,
+    ImageProperties,
+    PatientStatus,
+} from 'digital-fuesim-manv-shared';
+import {
+    Breathing,
+    Circulation,
+    Disability,
+    Exposure,
+    PatientHealthState,
+    PretriageInformation,
+    BiometricInformation,
+    PatientTemplate,
+} from 'digital-fuesim-manv-shared';
+
+export interface PatientData {
+    code: string;
+    templates: PatientTemplate[];
+}
+const defaultImage: ImageProperties = {
+    url: '/assets/patient.svg',
+    height: 80,
+    aspectRatio: 1,
+};
+
+export function parsePatientData(importString: string) {
+    const patientData: PatientData[] = [];
+
+    const splitString = importString.split('|');
```

Comment:
This applies to the one CSV File that we got I would like to discuss whether we should support more generic CSVs but most likely not in this PR